app/controllers/exports_controller.rb

```diff
@@ -9,12 +9,11 @@ class ExportsController < ApplicationController
   rescue_from Export::NotFound, with: :not_found
 
   def show
+    @page = params.fetch(:page, 0).to_i
     @workflow = export.workflow
-    if export.ancestor_purpose.present?
-      ancestor_result = @plate.ancestors.where(purpose_name: export.ancestor_purpose).first
-      locate_ancestor(ancestor_result.id) if ancestor_result.present?
-    end
-    render export.csv
+    @ancestor_plate = locate_ancestor
```

Comment:
I think the way I've set up the pipeline now, the blood array and blood bank plates should be the entry points to the Cardinal Library Prep and Cardinal Banking pipelines, so yes they should ideally be the stock and input plates.

Reply:
The flags won't work for tubes, and just get ignored by limber. 

I had a quick chat with Katy about this, and it was decided that the plates would both have the initial and stock flags set, and the submission would then be built from these. The transfer from the tubes to the plates is then tracked by the transfer requests only. This means that each plate is the stock to its own descendants, and as far as each library prep process is concerned is the start of the process.

There didn't seem to be much value in generating a request to go alongside that first transfer, as it wouldn't track any additional information, and would just delay the whole process. (Especially as you probably would want to build it in the plate creator, to avoid the need to create one per tube. Which means the creator would need to twiddle its thumbs while the submission built, only to almost immediately pass it)